content/pages/regler.mdx

```diff
@@ -2,9 +2,15 @@
 title: Regler
 description: fawekgnaejkgnewkøasg
 slug: /regler
-menu: 2
+menu: 3
 ---
 
 # Regler
-
-Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+  
+![The safe and inclusive space guidelines, made by Skeiv Verden. One, inclusion, everyone should be included, seen and weelcome. Two, pronouns, respect the pronoun people use. Ask if you don't know. Three, tone of communication, use respectful language, avoid jokes which can easily be misunderstood. Four, touching, ask before making physical contact. People have different needs for personal space. Five, flirting, be mindful of not crossing other people's boundaries, flirting can make some people uncomfortable. Six, privacy, everyone has a different story and can decide what they want to share about themselves. It's okay to say no. Seven, confidentiality, respect anonymity. Do not share personal information of others. Eight, cameras, photos should always be taken with consent. Nine, open mind, misunderstandings happen, take care of each other. Relax, everyone makes mistakes! Accept and or give feedback in a constructive way. And be kind, always!](../images/safespage.jpg)  
```

Comment:
Husk å skrive caption med copyright dersom du bruker noen andres bilde. Dette kan skrives på føldende format:

````
![Amazon Rainforest](/path/to/image)
<figcaption>The Amazon Rainforest contains a multitude of species.</figcaption>
````

Eller dersom du lenker til kilden: kan du gjøre det på én av to måter:


**Inline pure html**
````
![Big Red Train](/path/to/image)
<figcaption>The <a href="https://example.com">ABC Train</a> is <em>very</em> big and red.</figcaption>
````


**Med markdown lenke, med html på egne linjer**

````
![Big Red Train](/path/to/image)
<figcaption>
The [ABC Train](https://example.com) is *very* big and red.
</figcaption>
````


Eksemplene er hentet herfra: https://thesynack.com/posts/markdown-captions/

Reply:
Tror jeg gjorde den riktig nå, ser det bra ut?